src/app/components/mainContent/mainContent.component.tsx

```diff
@@ -1,42 +1,65 @@
 import { useStoreon } from 'storeon/react';
+import { useEffect } from "react";
 import { ContentItem } from '../../modules/content/content.state';
-import { ContentTile, ContentLabel } from '../contentItem/contentTile.component';
+import {
+    ContentTile,
+    ContentLabel,
+} from '../contentItem/contentTile.component';
 import './mainContent.component.scss';
+import { LoadingIndicator } from '../loadingIndicator/loadingIndicator.component';
 
-export function MainContent(): JSX.Element | null {
+export function MainContent(): JSX.Element {
     const { content } = useStoreon('content');
 
-    if (!content) return null;
-    return (
-        content && (
-            <div className="main-content-container">
-                <div className="articles-container">
-                    {content?.articles.length >= 3 &&
-                        content.articles
-                            ?.slice(0, 3)
-                            .map((article: ContentItem) => (
-                                <ContentTile contentItem={article} contentLabel={ContentLabel.Article} />
-                            ))}
-                </div>
+    useEffect(() => {
+      console.log(content.pending);  
```

Comment:
in general we should not use console.log in the app.

Reply:
console.log removed